PERF/cdm-v4.properties

```diff
@@ -20,8 +20,8 @@
 # Connections are made with the Spark Cassandra Connector; conventions there likely work here
 # See https://github.com/datastax/spark-cassandra-connector/blob/master/doc/reference.md
 #
-#  spark.cdm.origin.connect
```

Comment:
This doesn't adhere to the recommendation suggested here, https://github.com/datastax/cassandra-data-migrator/pull/163#pullrequestreview-1465143091